js/common/modules/@arangodb/test-helper.js

```diff
@@ -103,11 +102,8 @@ exports.waitForFoxxInitialized = function () {
       const coordinators = exports.getEndpointsByType('coordinator');
       let done = 0;
       coordinators.forEach((endpoint) => {
-        let res = request.get({
-          url: endpoint + '/wenn-der-fuxxmann-zweimal-klingelt',
-          timeout: 5
-        });
-        if (res.status === 404) {
+        let res = arango.GET_RAW('/wenn-der-fuxxmann-zweimal-klingelt');
```

Comment:
The previous implementation was able to send data to _multiple_ coordinators with multiple endpoints. 
The changed implementation now sends all requests to the same endpoint.

Reply:
Oops, you're right. This is broken now.